src/components/Home.js

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { filterWeather } from '../redux/home/home';
+import Location from './Location';
+import logo from '../assets/images/logo.svg';
+import logoLocation from '../assets/images/location.svg';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const cityList = useSelector((state) => state.homeReducer);
+  const handleChange = (e) => {
+    dispatch(filterWeather(e.currentTarget.value));
+  };
+  const [localWeather, setWeather] = useState('');
+  const [localTemp, setTemp] = useState('');
+  const [localLocation, setLocalLocation] = useState('Microverse');
+  const [localCoords, setLocalCoords] = useState('');
+  const handleLocalWeather = () => {
+    if ('geolocation' in navigator) {
+      navigator.geolocation.getCurrentPosition((position) => {
+        console.log('Latitude is :', position.coords.latitude);
+        console.log('Longitude is :', position.coords.longitude);
```

Comment:
- Let's clean up your code a little bit. As stated in the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), it is not good practice to commit `console.log()` as it’s ugly, kills performance and can make confidential data be visible to anyone. Kindly remove these console logs

Reply:
well addressed this!